about.html

```diff
@@ -12,57 +12,69 @@
 </head>
 
 <body>
-    <div class="container about-page">
+  <header>
+    <nav class="navBar">
+        <img src="img/lambda-black.png">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+    </nav>
+    <img src="img/jumbo-about.png">
+</header>
+    <div class="aboutcontainer">
      
-    About
+    <h3 class="topBottom">About</h3>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    <p class="aboutLine">Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
 
-
-    <img src="img/about-plan.png" alt="strategy">
-    
```

Comment:
The content area of the about page is a great place to use semantic elements like `<section>` and `<article>`

Reply:
Good looking out.